azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -928,7 +928,7 @@ public void cancelExecutingJobs(ExecutableFlow exFlow, String userId,
     }
   }
 
-  private void applyDisabledJobs(List<Object> disabledJobs,
+  public static void applyDisabledJobs(List<Object> disabledJobs,
```

Comment:
not sure whether the change is appropriate. A util class should be more suitable to be put.